README.md

```diff
@@ -48,7 +50,7 @@ The AC calculations are based on full Newton-Raphson algorithm. The DC calculati
 
  Open Load Flow supports:
  - Generator and static var compensator voltage remote control through PQV bus modelling. It supports any kind of shared voltage control between controllers that can be generators, static var compensators, or VSC converter stations.
- - Static var compensator local voltage control with a slope (support the powsybl-core extension [```VoltagePerReactivePowerControl```](https://www.powsybl.org/pages/documentation/grid/model/extensions.html).
+ - Static var compensator local voltage control with a slope (support the powsybl-core extension [```VoltagePerReactivePowerControl```](https://powsybl.readthedocs.io/projects/powsybl-core/en/latest/grid_model/extensions.html).
```

Comment:
Here, if you want, you can use the anchor to redirect directly to the extension:
https://powsybl.readthedocs.io/projects/powsybl-core/en/latest/grid_model/extensions.html#voltage-per-reactive-power-control
(I think it should work)